Stanford/JumpstartEngineering/Install/Layouts/Sitewide.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @file
+ * Configure the sitewide layout.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Layouts;
+/**
+ *
+ */
+class Sitewide extends \AbstractInstallTask {
+
+  /**
+   * Configure the sitewide layout.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+
+    $context_status = variable_get('context_status', array());
```

Comment:
Please add a comment to what is going on here. I think it is disabling the sitewide_jsa context with comes bundled in the jsa_layouts feature and replacing it with a jse context. Why does jse need to be enabled? Did someone disable it in the feature for a reason?